README.md

```diff
@@ -11,8 +12,6 @@
   - [Prerequisites](#prerequisites)
   - [Install](#install)
   - [Usage](#usage)
-  - [Run tests](#run-tests)
-- [👥 Authors](#authors)
```

Comment:
- [ ] As mentioned by the previous reviewer you should fix the `Run tests` link or delete it and not the `Authors` link because you have a dedicated section for `Authors` which means you should have it in your Table of contents. Also, I will suggest keeping the `Run tests` link instead of deleting it because you have tests in your project and a section `Testing` in your README file, you should have a dedicated link for tests in the table of contents 🙃 🙃

Reply:
- Well resolved 👍🏽